benchmark/fs/readfile-partitioned.js

```diff
@@ -17,12 +17,13 @@ const zlib = require('zlib');
 const assert = require('assert');
 
 const bench = common.createBenchmark(main, {
-  dur: [5],
+  duration: [5],
+  encoding: ['undefined', 'utf-8'],
```

Comment:
are you sure `undefined` should be a string?

Reply:
This is intentional. There is a particular check in benchmark/common.js line 121 which prohibits usage of undefined:

```
/Users/yagiz/Developer/node/benchmark/common.js:120
          throw new TypeError(
          ^

TypeError: configuration "encoding" had type undefined
    at recursive (/Users/yagiz/Developer/node/benchmark/common.js:120:17)
    at recursive (/Users/yagiz/Developer/node/benchmark/common.js:132:11)
    at Benchmark._queue (/Users/yagiz/Developer/node/benchmark/common.js:140:7)
    at new Benchmark (/Users/yagiz/Developer/node/benchmark/common.js:35:23)
    at Object.createBenchmark (/Users/yagiz/Developer/node/benchmark/common.js:409:12)
    at Object.<anonymous> (/Users/yagiz/Developer/node/benchmark/fs/readfile.js:16:22)
    at Module._compile (node:internal/modules/cjs/loader:1119:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1173:10)
    at Module.load (node:internal/modules/cjs/loader:997:32)
    at Module._load (node:internal/modules/cjs/loader:838:12)
```

Reply:
Why not just use `''` instead of `'undefined'`? We already use empty strings for some parameters in other benchmarks.

Reply:
Thanks for the review. I force pushed and updated the commit. Appreciate if you can review it again